public/usage-examples/graphics/current_clip/current-clip-1-simple.cpp

```diff
@@ -0,0 +1,24 @@
+#include "splashkit.h"
+
+int main() 
+{
+    // Open a window with the title "Current Clip" and dimensions
+    window win = open_window("Current Clip", 800, 600);
```

Comment:
Should use window here to make sure it keep the same as the other language code.